Dockerfile-scraper

```diff
@@ -0,0 +1,8 @@
+FROM alpine:latest
+
+ADD --chmod=755 bin/nri-kubernetes-linux-amd64-scraper /var/db/newrelic-infra/newrelic-integrations/bin/
+
+# creating the nri-agent user used only in unprivileged mode
+RUN addgroup -g 2000 nri-agent && adduser -D -u 1000 -G nri-agent nri-agent
```

Comment:
I don't see any reason for the scraper (`nri-kubernetes`) to not be unprivileged by default, as it shouldn't need to mount anything. We could then change the `USER` here to the newly created one and save some redundant files in the manifests :)

Reply:
Perhaps we coudl rename this user to something that is the same in both images, e.g. `newrelic`.

Reply:
The user is still root, I am just creating the user, not setting it up.

We need to add it to have the proper name when running the image in unprivileged with "runAsUser 1000"

Reply:
> The user is still root, I am just creating the user, not setting it up.

This is exactly what I wanted to say, why not switch the user in the dockerfile itself?

Reply:
oh, ahhh, oh. Let me try

Reply:
We would lose some metrics 
 - https://docs.newrelic.com/docs/infrastructure/install-infrastructure-agent/linux-installation/linux-agent-running-modes/

Reply:
Sure, I added it!